docs/crocks/Arrow.md

```diff
@@ -0,0 +1,77 @@
+# Arrow
+
+`Arrow a b`
+
+--
+
+--
+
+```js
+--
+```
+
+`Arrow` exposes the following functions on the constructor and instance:
+
+| Constructor | Instance |
+|:---|:---|
+| [`empty`](#empty), [`type`](#type) | [`both`](#both), [`concat`](#concat), [`contramap`](#contramap), [`empty`](#empty), [`first`](#first), [`inspect`](#inspect), [`map`](#map), [`promap`](#promap), [`runWith`](#runWith), [`second`](#second), [`type`](#type), [`value`](#value) |
+
+## Constructor
+
+### empty
+
+`Arrow m => () -> m () b`
```

Comment:
The identity or `empty` for `Arrow` is an identity function, so this should be:

`Arrow m => () -> m a a`